Tests/NostrSDKTests/Fixtures/ok_unknown_reason.json

```diff
@@ -0,0 +1 @@
+["OK", "b1a649ebe8b435ec71d3784793f3bbf4b93e64e17568a741aecd4c7ddeafce30", true, "unknown: reason: unknown"]
```

Comment:
Is it possible that the message will have two colons in it?

Reply:
Yes, it's possible. Technically the relay can give us whatever it wants and not conform to NIP-01. I wanted to stress test that possibility.